src/pyOpenMS/pxds/DigestionEnzymeProtein.pxd

```diff
@@ -25,61 +25,61 @@ cdef extern from "<OpenMS/CHEMISTRY/DigestionEnzymeProtein.h>" namespace "OpenMS
                                UInt omssa_id) nogil except +
 
         # sets the name of the Enzyme
-        void setName(String name) nogil except +
+        void setName(String name) nogil except + # wrap-doc:Sets the name of the Enzyme
```

Comment:
```suggestion
        void setName(String name) nogil except + # wrap-doc:Sets the name of the enzyme
```

Reply:
This might lead to collisions if https://github.com/OpenMS/OpenMS/pull/5396 is merged.